src/coponents/Permission.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useState, useEffect} from "react";
+import { View, Text, StyleSheet } from "react-native";
+import ToggleButton from "./ToggleButton";
+import {Colors} from '../utils/colors'
+import axios from 'axios';
+import { useUserContext } from '../context/UserContext';
+import {PERMISSION_URL} from '../utils/api'
+export default function Permission(){
+    const [permission, setPermission] = useState(["Step Count", "Sleep Hours", "Heart Rate", "Notification"]);
+    const [enabled, setEnabled] = useState({})
```

Comment:
It would be really helpful if you added a type for the enabled variable. It's hard to predict what this variable means.